spec/requests/npq_separation/admin_controller_spec.rb

```diff
@@ -3,9 +3,11 @@
 RSpec.describe NpqSeparation::AdminController, type: :request do
   include Helpers::NPQSeparationAdminLogin
 
+  subject(:admin_response) { get(npq_separation_admin_path) && response }
```

Comment:
I personally would name it `admin_request`, just feels better for me.

Reply:
Yeah, thats what I've usually been naming this. 

The reason I decided that was the wrong name was it doesn't actually return a page request object, it returns a page response object - it makes a request, but this is equivalent to a `let` and you'd name a `let` for what it holds, not how it got it, eg `let(:user)` instead of `let(:create_factory_user)`

So its either `make_request` (ie named for what its doing) or `admin_response` (ie named for what it is) ?